form-flow/src/main/kotlin/com/ritense/formflow/security/ValtimoFormFlowHttpSecurityConfigurer.kt

```diff
@@ -1,20 +1,20 @@
 /*
- *  Copyright 2015-2024 Ritense BV, the Netherlands.
```

Comment:
?

Reply:
Looks like automatic reformat of the license because of extra whitespace.